src/Features/CSharpTest/ConvertLinq/ConvertForEachToLinqQueryTests.cs

```diff
@@ -4156,10 +4156,10 @@ List<int> M(IEnumerable<int> nums)
                         return /*30*/ /* 1 *//* 2 *//* 3 *//* 4 */// 5
                        /*31*//* 6 */
                        (from/* 8 *//* 7 *//* 9 */x /* 10 */ in/* 11 */nums/* 12 */// 13
-                                  /* 14 */// 15
-                                  /* 16 *//* 17 */
-                              let y /* 18 */ = /* 19 */ x + 1/* 20 *///21
-                              select y)/* 24 *//*27*///28
+                            /* 14 */// 15
+                            /* 16 *//* 17 */
+                        let y /* 18 */ = /* 19 */ x + 1/* 20 *///21
+                        select y)/* 24 *//*27*///28
```

Comment:
these results are expected.  the other clauses align with the `from` now.

Reply:
aside; these tests are insane.  they don't reflect any sort of real code, and it's nigh impossible to reason about them.